snapcraft.yaml

```diff
@@ -32,36 +32,37 @@ parts:
   gnome-logs:
     source: https://gitlab.gnome.org/GNOME/gnome-logs.git
     source-type: git
-    source-tag: gnome-logs-3.36.0
+    source-tag: 'gnome-logs-42.0'
     plugin: meson
-    meson-parameters: [--prefix=/usr]
-    organize:
-      snap/gnome-logs/current/usr: usr
+    parse-info: [usr/share/metainfo/org.gnome.Logs.appdata.xml]
+    meson-parameters:
+      - --prefix=/usr
     build-packages:
       - libsystemd-dev
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10 | sed -e s/gnome-logs-//g)
+      craftctl default
+      craftctl set version=$(git describe --tags --abbrev=10 | sed -e s/gnome-logs-//g)
       sed -i.bak -e 's|=org.gnome.Logs$|=${SNAP}/meta/gui/org.gnome.Logs.svg|g' data/org.gnome.Logs.desktop.in
       sed -i.bak -e 's|symlink_media: true|symlink_media: false|g' help/meson.build
     override-build: |
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp ../src/data/icons/scalable/org.gnome.Logs.svg $SNAPCRAFT_PART_INSTALL/meta/gui/
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/scalable/org.gnome.Logs.svg $CRAFT_PART_INSTALL/meta/gui/
 
   libraries:
+    after: [ gnome-logs ]
     plugin: nil
     prime:
-      - "-*"
+      - -"*"
```

Comment:
So you want to unprime literally everything in the prime dir? How does this work to give us a working app?

Reply:
Mmm... good point... But what is the difference between both statements?

Reply:
In fact... why is there that part? I think it does nothing...

Reply:
Indeed, just drop the libraries part completely.  This is leftover from when we had been staging packages as depends.

Reply:
Fixed.